examples/2d/elixir_euler_blob_amr.jl

```diff
@@ -0,0 +1,78 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the compressible Euler equations
+gamma = 5/3
+equations = CompressibleEulerEquations2D(gamma)
+
+initial_conditions = initial_conditions_blob
+
+surface_flux = flux_lax_friedrichs
+volume_flux  = flux_chandrashekar
+basis = LobattoLegendreBasis(4)
+
+indicator_sc = IndicatorHennemannGassner(equations, basis,
+                                         alpha_max=0.4,
+                                         alpha_min=0.0001,
+                                         alpha_smooth=true,
+                                         variable=pressure)
+
+volume_integral = VolumeIntegralShockCapturingHG(indicator_sc;
+                                                 volume_flux_dg=volume_flux,
+                                                 volume_flux_fv=surface_flux)
+
+solver = DGSEM(basis, surface_flux, volume_integral)
+
+coordinates_min = (-20, -20)
+coordinates_max = ( 20,  20)
+
+mesh = TreeMesh(coordinates_min, coordinates_max,
+                initial_refinement_level=6,
+                n_cells_max=100_000,)
+
+semi = SemidiscretizationHyperbolic(mesh, equations, initial_conditions, solver)
+
+
+###############################################################################
+# ODE solvers, callbacks etc.
+
+tspan = (0.0, 8.0)
+ode = semidiscretize(semi, tspan)
+
+summary_callback = SummaryCallback()
+
+amr_indicator = IndicatorHennemannGassner(semi,
+                                          alpha_max=1.0,
+                                          alpha_min=0.0001,
+                                          alpha_smooth=true,
+                                          variable=pressure)
+amr_controller = ControllerThreeLevel(semi, amr_indicator,
+                                      base_level=4,
+                                      max_level =7, max_threshold=0.01)
```

Comment:
It looks like the old AMR indicator used some different thresholds.
```julia
  elseif dg.amr_indicator === :blob
    base_level = 4
    max_level = 7
    super_max_level = 7
    blending_factor_threshold0 = 0.3
    blending_factor_threshold1 = 0.003
    blending_factor_threshold2 = 0.0003
```
Did you compare the results?

You can also add a `med_level` and `med_threshold` for the `ControllerThreeLevel`. See `?ControllerThreeLevel` in the Julia REPL.

Reply:
Okay, I have activated the three level controller for the blob test